### ISSUE TITLE
Type Error "got an unexpected keyword argument 'prompt_required'" 

### ISSUE THREAD

**Issue by danielsteiner** [mvt-project/mvt#45]
Hi, i just cloned the repo and installed MVT v 1.0.11. 
Sadly, I get following error 

```
Traceback (most recent call last):
  File "/home/user/.local/bin/mvt-ios", line 5, in <module>
    from mvt.ios import cli
  File "/home/user/.local/lib/python3.8/site-packages/mvt/ios/__init__.py", line 6, in <module>
    from .cli import cli
  File "/home/user/.local/lib/python3.8/site-packages/mvt/ios/cli.py", line 53, in <module>
    def decrypt_backup(destination, password, key_file, backup_path):
  File "/usr/lib/python3/dist-packages/click/decorators.py", line 173, in decorator
    _param_memo(f, OptionClass(param_decls, **option_attrs))
  File "/home/user/.local/lib/python3.8/site-packages/mvt/common/options.py", line 23, in __init__
    super(MutuallyExclusiveOption, self).__init__(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1547, in __init__
    Parameter.__init__(self, param_decls, type=type, **attrs)
TypeError: __init__() got an unexpected keyword argument 'prompt_required'
```

has anyone got an idea how to fix that or how to get it running?

**Comment by edruca** [comment]
Happened same to me!, i'm using the last version of Kali Linux

**Comment by goshawk22** [comment]
Try removing ` prompt_required=False,` from this line. https://github.com/pkirkovsky/mvt/blob/695555f26f61bd79cc191b1a6e7c213043980c3f/mvt/ios/cli.py#L46

**Comment by pascal-gujer** [comment]
Same here, got around it by using the „prebuilt“ version (pip3 install mit)
Unfortunately https://github.com/mvt-project/mvt/pull/31#issue-693891310 is present in the pip3 Version…

any ideas?

**Comment by pascal-gujer** [comment]
> Try removing ` prompt_required=False,` from this line. https://github.com/pkirkovsky/mvt/blob/695555f26f61bd79cc191b1a6e7c213043980c3f/mvt/ios/cli.py#L46

Seems to work so far, no errors. Thanks!

just checked a DUT and got only infos, so far this device seems to be not affected.

**Comment by edruca** [comment]
Thanks, but it doesn't work for me, continue getting:
`Traceback (most recent call last):
  File "/home/edu/.local/bin/mvt-ios", line 5, in <module>
    from mvt.ios import cli
  File "/home/edu/.local/lib/python3.9/site-packages/mvt/ios/__init__.py", line 6, in <module>
    from .cli import cli
  File "/home/edu/.local/lib/python3.9/site-packages/mvt/ios/cli.py", line 53, in <module>
    def decrypt_backup(destination, password, key_file, backup_path):
  File "/usr/lib/python3/dist-packages/click/decorators.py", line 192, in decorator
    _param_memo(f, OptionClass(param_decls, **option_attrs))
  File "/home/edu/.local/lib/python3.9/site-packages/mvt/common/options.py", line 23, in __init__
    super(MutuallyExclusiveOption, self).__init__(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1714, in __init__
    Parameter.__init__(self, param_decls, type=type, **attrs)
TypeError: __init__() got an unexpected keyword argument 'prompt_required'`

**Comment by pkirkovsky** [comment]
@edruca Could you please show which command you're executing and how it's structured?
Also, please provide the output of `pip3 show click`? Mine shows `Version: 8.0.1`

**Comment by kris71** [comment]
I have the exact same issue. I was in version 7.0 of click. I ran `pip install -U click` to get version 8.0.1 and it fix the issue

**Comment by edruca** [comment]
> I have the exact same issue. I was in version 7.0 of click. I ran `pip install -U click` to get version 8.0.1 and it fix the issue

That was the problem, i had the 7.1.2 version, now i have 8.0.1 and all works,
thanks to all!!!!

**Comment by pkirkovsky** [comment]
Specifying the latest version of `click` should fix this for everyone going forward: #84
